src/ucp/tag/eager_snd.c

```diff
@@ -212,6 +213,7 @@ const ucp_proto_t ucp_tag_eager_proto = {
     .zcopy_single            = ucp_tag_eager_zcopy_single,
     .zcopy_multi             = ucp_tag_eager_zcopy_multi,
     .zcopy_completion        = ucp_tag_eager_zcopy_completion,
+    .start_rndv              = ucp_proto_start_rndv,
```

Comment:
To make rndv protocol consistent with other ones by design. When I tried to modify the protocol (disabling of sync_send), it was a problem.